README.md

```diff
@@ -1,78 +1,65 @@
+We created this repository in 2016 to accept plugin submissions from the Grafana community. 5 years and more than 150 published plugins later, it's clear that community plugins have been instrumental in the success of Grafana. We'd like to make the process of submitting plugins even easier.
 
-We review all plugins before they are published. This means that it may take some time before we can review your plugin.
+We're excited to announce that you can now [submit your plugin](#submit-your-plugin) and [maintain your plugin](#maintain-your-plugin) directly from your Grafana Cloud account!
 
-Here's a few things you can do to help us review your plugin faster.
+This also means that we're starting the process of deprecating this repository. Here's how it affects you:
 
-- Validate your plugin release with [this plugin validator](http://plugin-validator.grafana.net)
-- Use the [GitHub workflows](https://github.com/grafana/plugin-workflows) to automate your plugin release
-- Read the [Review Guidelines](http://docs.grafana.org/plugins/developing/plugin-review-guidelines/) before submitting your plugin. These guidelines determine if the plugin is ready to be published or not.
-- If possible, for datasource plugins please provide a description on how to set up a simple test environment. A docker container or simple install script helps speed up the review process a lot.
+- If your pull request has been triaged by us already and is waiting for review, you don't need to do anything. We'll continue to review the remaining plugin submissions in the pipeline.
+- If you'd like, you can migrate your current plugin submission by submitting it through the new submission flow. Let us know that you'd like to use the new flow by updating your pull request. Please keep the pull request open to keep your position in the review queue.
+- If you're submitting a new plugin or an update to an existing plugin, please use the new submission flow. We won't accept any new plugin submissions on this repository.
 
-To submit a plugin for review:
+If you have any questions, [create a new issue](https://github.com/grafana/grafana-plugin-repository/issues/new), or send an email to  `plugin-reviews@grafana.com`.
 
-1. Fork this repository
-1. Add your plugin to `repo.json`
-1. Create a pull request
+## Submit your plugin
 
-> **Note:** Commercial plugins require a plugin subscription to be published. Commercial plugin subscriptions help us fund continued development of our open source platform and software. See the [terms](https://grafana.com/terms) for more details.
+Before you submit your plugin, we ask that you read our [Guidelines](#guidelines) and [Frequently Asked Questions](#faq).
```

Comment:
```suggestion
Before you submit your plugin, we ask that you read our [guidelines](#guidelines) and [frequently asked questions](#faq).
```
This is a style issue that we have not decided on as a team; do we use plain text that is close enough to the target text, or do we use WYSIWYG for link text. If the later, then "FAQ" is what we need to go with.